### DISCUSSION TITLE
[Terminology] Carbon Delta and Carbon Counterfactual

### DISCUSSION THREAD

**Discussion by TaylorPrewitt** [Green-Software-Foundation/sci#65] [General]
There has been a few different interpretations of this terminology, so I wanted to propose a couple definitions to get everyone on the same page. 

**Carbon delta**: The difference in emissions between a carbon aware and an unaware action.  
**Carbon counterfactual**: The *carbon delta* had the carbon aware action been different.

**Comment by atg-abhishek** [comment]
I believe this is now included in the SCI spec @TaylorPrewitt - please let me know if you concur and we can close this discussions. Thanks!
